includes/admin/downloads/metabox.php

```diff
@@ -416,11 +416,15 @@ function edd_render_price_row( $key, $args = array(), $post_id, $index ) {
 	// Run our advanced settings now, so we know if we need to display the settings.
 	// Output buffer so that the headers run, so we can log them and use them later
 	ob_start();
-	do_action( 'edd_download_price_table_head', $post_id );
+	if ( has_action( 'edd_download_price_table_head' ) ) {
+		do_action_deprecated( 'edd_download_price_table_head', array( $post_id ), '2.8', 'edd_download_price_option_row' );
```

Comment:
I wonder if we should change the version to `2.10`. Although it was technically intended to be deprecated in `2.8`, we didn't actually enforce that until now (`2.10`).